### DISCUSSION TITLE
How to properly set Backdrop-filter style in ui.dialog()?

### DISCUSSION THREAD

**Discussion by happybeginning1** [zauberzeug/nicegui#3302] [Q&A] [answered]
### Question

I want to use the backdrop-filter property in a ui.dialog(). I referred to Quasar's documentation (https://quasar.dev/vue-components/dialog) and used the following property settings:

```python
with ui.dialog(value=True).props('backdrop-filter="blur(5px) grayscale(80%) contrast(50%)"') as dialog:
    ui.label('testing')
```

However, the backdrop-filter properties don't seem to take effect.
Could you please advise on how to correctly set these properties?
Thank you very much!

**Comment by python-and-novella** [comment, marked as answer]
It needw quasar v2.15+,current quasar is 2.13.

**Reply by happybeginning1** [reply]
Oops, sorry I did notice this before. Thank you!
